### DISCUSSION TITLE
Is there a way to set the default admin password for a VM through Tart?

### DISCUSSION THREAD

**Discussion by sfc-gh-jubui** [cirruslabs/tart#635] [Q&A] [answered]
title

**Comment by fkorotkov** [comment, marked as answer]
If you are using Tart Packer plugin to create VMs from scratch please check [our templates where the username/password pair is configured](https://github.com/cirruslabs/macos-image-templates/blob/47695a9abdb5f07835eaf6629d9050f6f5ea2bc2/templates/vanilla-sonoma.pkr.hcl#L42-L43).

**Reply by sfc-gh-jubui** [reply]
Thank you! Exactly, what I was looking for.
